README.md

```diff
@@ -119,6 +125,15 @@ TODO
 
 TODO
 
+## Cluster usage
+
+### Useful `slurm` commands
```

Comment:
Since you started, maybe we should properly document the main commands? sbatch, squeue, sinfo, scontrol?

Reply:
Maybe we should point to the CECI documentation, this is quite well done imho https://support.ceci-hpc.be/doc/_contents/QuickStart/SubmittingJobs/SlurmTutorial.html

Reply:
Yes, we could add a pointer to that.

Then have some short explanations regarding how to request GPUs?

Reply:
As the script I have made will eventually become a command, I have removed the squeue example and added links to CECI guide and slurm documentation for a handful of commands (+ a note about gpu request)